.eslintrc.js

```diff
@@ -19,11 +19,8 @@ module.exports = {
   },
   plugins: ["@typescript-eslint", "prettier"],
   rules: {
-    indent: ["error", 2, { SwitchCase: 1 }],
     "linebreak-style": ["error", "unix"],
     "no-console": "error",
-    quotes: ["error", "double"],
-    semi: ["error", "never"],
```

Comment:
These rules can be safely removed as they are enforced by the `prettier` plugin.